docs/serving/autoscaling.md

```diff
@@ -157,15 +160,15 @@ a revision is not referenced by any route, the minimal pod count specified by
 `minScale` will still be provided. Keep in mind that non-routeable revisions may
 be garbage collected, which enables Knative to reclaim the resources.
 
-### Default behavior
+#### Default behavior
 
 If the `minScale` annotation is not set, pods will scale to zero (or to 1 if
 `enable-scale-to-zero` is `false` per the ConfigMap mentioned above).
 
 If the `maxScale` annotation is not set, there will be no upper limit for the
 number of pods created.
 
-## Configuring CPU-based autoscaling
+### Configuring CPU-based autoscaling
```

Comment:
Note for future work: This should be another top level section that says:

```
# Configuring Horizontal Pod Autoscaler (HPA)
```

This section should then talk about how Knative Serving can also scale using the HPA, which enables CPU based metrics (as shown here) or also use concurrency metrics which we use for the KPA as well. Let's split that off though.

Reply:
If you want to create an issue for this maybe I'd be happy to work on it next? :smiley_cat:

Reply:
Updated the heading for now - lmk if you want to follow up the 'splitting off' part in another issue